apps/docs/pages/guides/database/extensions/pg_stat_monitor.mdx

```diff
@@ -0,0 +1,84 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'pg_stat_monitor',
+  title: 'pg_stat_monitor: Query Performance Monitoring',
+  description: 'Query performance monitoring for PostgreSQL',
+}
+
+[pg_stat_monitor](https://github.com/percona/pg_stat_monitor) is query performance monitoring tool that provides query execution statistics in a SQL view named `pg_stat_monitor`. It tracks a superset of statistics available in [`pg_stat_statements`](https://www.postgresql.org/docs/current/pgstatstatements.html).
```

Comment:
Suggest noting that pg_stat_monitor does not persist data across server restarts, and also that it tracks data for a moving time window (older data gets discarded).